entrypoint.sh

```diff
@@ -202,7 +204,7 @@ configure_nvidia_installation_dirs() {
   update_container_ld_cache
 
   # Install an exit handler to cleanup the overlayfs mount points.
-  trap "{ umount /lib/modules/\"$(uname -r)\"/video; umount /usr/lib/x86_64-linux-gnu ; umount /usr/bin; }" EXIT
+  trap '{ umount /lib/modules/"$(uname -r)"/video; umount /usr/lib/x86_64-linux-gnu ; umount /usr/bin; }' EXIT
```

Comment:
`trap` should refer to a function... It's a lot more readable and doesn't have to worry about crazy escaping of characters.

Reply:
I am with you, but I dont want to differ much from upstream